bindings/c/test/mako/async.cpp

```diff
@@ -133,8 +136,11 @@ void ResumableStateForRunWorkload::runOneTick() {
 					                       err.what());
 					updateErrorStats(err, iter.op);
 					tx.onError(err).then([this, state = shared_from_this()](Future f) {
-						const auto rc = handleForOnError(
-						    tx, f, fmt::format("{}:{}", iter.opName(), iter.step), args.transaction_timeout > 0);
+						const auto rc =
+						    handleForOnError(tx,
+						                     f,
+						                     fmt::format("{}:{}", iter.opName(), iter.step),
+						                     args.transaction_timeout_tx > 0 || args.transaction_timeout_db > 0);
```

Comment:
could you introduce a new member function to `Arguments`? This is too lengthy. `Argument::isAnyTimeoutEnabled()` or similar.

Reply:
Done with the latest commit.